src/charm.py

```diff
@@ -249,6 +249,22 @@ def enable_host_network(self) -> bool:
         """Get the hostNetwork enabling of csi-*plugin-provisioner deployments."""
         return bool(self.config.get("enable-host-networking"))
 
+    @property
+    def kubelet_dir(self) -> str:
+        """Get the kubelet directory from the kubernetes-info relation (if available)"""
+        try:
+            relation = self.model.get_relation("kubernetes-info")
+            return relation.data[relation.app]["kubelet_dir"]
```

Comment:
it's dumb, but my only ask is that this be based on the [upstream argument](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/) 

>  --root-dir string     Default: /var/lib/kubelet
> --
>   Directory path for managing kubelet files (volume mounts, etc).


```suggestion
            return relation.data[relation.app]["kubelet-root-dir"]
```